packages/mobile/src/tokens/selectors.ts

```diff
@@ -102,10 +102,15 @@ export const defaultTokenToSendSelector = createSelector(
 )
 
 export const totalTokenBalanceSelector = createSelector(
-  [tokensWithUsdValueSelector, localCurrencyExchangeRatesSelector, tokenFetchErrorSelector],
-  (tokensWithUsdValue, exchangeRate, tokenFetchError) => {
+  [
+    tokensListSelector,
+    tokensWithUsdValueSelector,
+    localCurrencyExchangeRatesSelector,
+    tokenFetchErrorSelector,
+  ],
+  (tokensList, tokensWithUsdValue, exchangeRate, tokenFetchError) => {
     const usdRate = exchangeRate[Currency.Dollar]
-    if (!usdRate) {
+    if (!usdRate || Object.keys(tokensList).length === 0) {
```

Comment:
Isn't `tokensList` an array? I think we should check `tokensList.length` instead of `Object.keys().length`

Reply:
yikes, i don't know why i thought i was working with the `tokens` directly. i updated this, thanks for catching!